src/FixedDataTable.js

```diff
@@ -486,6 +486,7 @@ var FixedDataTable = React.createClass({
           scrollLeft={state.scrollX}
           fixedColumns={state.groupHeaderFixedColumns}
           scrollableColumns={state.groupHeaderScrollableColumns}
+          maxVisibleColumns={state.maxVisibleColumns}
```

Comment:
I don't have a good answer :/

We might be able to get away without this entirely though.  How would you feel about having the element array in CellGroup be allowed to grow, but never shrink?